test/end-to-end/DrawerLogin.test.ts

```diff
@@ -157,10 +157,6 @@ test.before(async (t) => {
   // login to the transfer email to activate it, then logout
   await LoginProcedure(t, transferEmail)
 
-  if (await userModal.exists) {
-    await t.click(userModalCloseButton)
-  }
-
```

Comment:
nit: would you mind removing the imports for these as well! 🙇

Reply:
done